dataikuapi/dss/apiservice.py

```diff
@@ -106,3 +106,16 @@ def download_package_to_file(self, package_id, path):
                     f.write(chunk)
                     f.flush()
 
+    def publish_package(self, package_id, published_service_id=None):
+        """
+        Publish a package on the API Deployer.
+
+        :param string package_id: The identifier of the package
+        :param string published_service_id: The identifier of the API service on the API Deployer where the package will be published.
+            A new published API service will be created if none matches the identifier.
+            If the parameter is not set, the identifier from the current :class:`DSSAPIService` is used.
+        """
+        params = None
+        if published_service_id is not None:
+            params = {"publishedProjectKey": published_service_id}
+        return self.client._perform_json("POST", "/projects/%s/apiservices/%s/packages/%s/publish" % (self.project_key, self.service_id, package_id), params=params)
```

Comment:
This publish api service call doesn't return any JSON like the `publish_bundle` version, so you should do `_perform_empty` instead.

Reply:
Oh you're right, will remove the return as well

Reply:
Although side note, maybe in the long run we'll have this call return something, if we want to have links to the published service in the package list of the API Designer (similar to what we have in the bundle list for now) : https://app.clubhouse.io/dataiku/story/8554/show-which-versions-are-already-pushed-and-replace-by-a-link-that-takes-you-to-the-api-deployer-with-a-preselected-filter